deftree/deftree.go

```diff
@@ -339,6 +339,15 @@ func (self *ProtoEnum) Describe(depth int) string {
 	return rv
 }
 
+func (pe *ProtoEnum) GetByName(name string) Describable {
```

Comment:
This is not constistant with the reciever name as the methods above it. 

Above they are:

```
func (self *ProtoEnum) ...
```

I like changing it from `self` to `pe` though please update all of them to `pe` if that is the change.